### ISSUE TITLE
How to add modules in stancl/tenancy?

### ISSUE THREAD

**Issue by abbasmashaddy72** [nWidart/laravel-modules#1488]
I'm also working with this package, how can I implement stancl/tenancy with laravel-modules.

This Question was also asked previously.

Reference:
#https://github.com/nWidart/laravel-modules/issues/1300
#https://github.com/nWidart/laravel-modules/issues/1250
#https://github.com/nWidart/laravel-modules/issues/1238
#https://github.com/nWidart/laravel-modules/issues/1111
#https://github.com/nWidart/laravel-modules/issues/882

But reply are not giving the exact implementation methods clear.

It will very help full if someone gives a small project implementing it or give a clear instruction how to implement it step by step.

Thanks in advance.

If anyone not familiar with stancl/tenancy you can show with some other package but with multi DB & subdomains.

Sorry, For Including Many Users

@zoxed @wyklif @wikigods @ducorteam @Vestibule @ctf0 @Surt @wuwx @dydx @jnbn @Schweppesale @armababy

**Comment by Surt** [comment]
I think that the problem resides in stancl/tenancy it is not a "drop-in" package, you should change all your migrations to accomodate to them. (not to talk about publishing all migrations of the packages you are using)

To avoid that what i do is ```php artisan migrate --path database/migrations/for-central```  on my "myapp:install" command, so the central (tenancy) database is created correcly. Then nWidart/laravel-modules take care of the migration for modules everytime i created a tenant.

So, in short:

1. custom command to migrate only the published "tenancy" migrations
2. publish all the migrations for third party packages on my database/migrations inside my "main" module
3. all done, each time a tenant is created the tenant database is created with my normal migrations + the published ones

**Comment by giagara** [comment]
> Thanks, @wikigods
> 
> I will try, and I will share the git project link

I'm interest in it too. Did you create a project you want to share?

**Comment by anburocky3** [comment]
I too looking for it. It will be helpful for most people/

**Comment by mattiadelfranco** [comment]
Hi, sorry to open this, but I need help on getting the laravel-modules migrations executed when I create a new Tenant, actually it doesn't
